cli/common/tarantool.go

```diff
@@ -14,11 +14,92 @@ import (
 )
 
 var (
-	tarantoolVersionRegexp *regexp.Regexp
+	tarantoolVersionRegexp      *regexp.Regexp
+	tarantoolVersionFullRegexp  *regexp.Regexp
```

Comment:
Well, the main reason is to change only what we need to change to solve the issue and do not rework whole module (at least in this PR). I've tried to add some comments, but not sure it will be a satisfying answer.

Reply:
Ok. I think then we should create an issue with the `code health` tag to rework the whole module on new parsing when this PR is merged.

Reply:
https://github.com/tarantool/cartridge-cli/issues/665